androidHyperskillApp/src/main/java/org/hyperskill/app/android/placeholder_new_user/fragment/PlaceholderNewUserFragment.kt

```diff
@@ -53,12 +52,8 @@ class PlaceholderNewUserFragment :
 
     private var viewDataMapper: PlaceholderNewUserViewDataMapper? = null
 
-    private val svgImageLoader by lazy(LazyThreadSafetyMode.NONE) {
-        ImageLoader.Builder(requireContext())
-            .components {
-                add(SvgDecoder.Factory())
-            }
-            .build()
+    private val svgImageLoader: ImageLoader by lazy(LazyThreadSafetyMode.NONE) {
+        HyperskillApp.graph().imageLoadingComponent.imageLoader
```

Comment:
Why not inject inside of the `injectComponents` function? and in all other Fragments

Reply:
In this case dependency could be received lazily, cause it isn't dependent on lifecycle.
Also, it's the way to not use lateinit vairable.